lib/commands/general.js

```diff
@@ -97,6 +101,25 @@ commands.getWindowSize = async function getWindowSize (windowHandle = 'current')
   }
 };
 
+/**
+ * Retrieves the current device's timestamp.
+ *
+ * @param {string} format - The set of format specifiers. Read
+ *                          https://momentjs.com/docs/ to get the full list of supported
+ *                          datetime format specifiers. The default format is
+ *                          `YYYY-MM-DDTHH:mm:ssZ`, which complies to ISO-8601
+ * @returns Formatted datetime string or the raw command output if formatting fails
+ */
+commands.getDeviceTime = async function getDeviceTime (format = MOMENT_FORMAT_ISO8601) {
+  log.info('Attempting to capture iOS device date and time');
+  if (this.isRealDevice()) {
+    const { timestamp, utcOffset } = await utilities.getDeviceTime(this.opts.udid);
+    return moment.unix(timestamp).utcOffset(utcOffset).format(format);
```

Comment:
should we convert utcOffset to minutes here or it is already converted inside ios-device?

Reply:
It is already converted in appium-ios-device